docs/HowTo/develop-fprime-libraries.md

```diff
@@ -0,0 +1,75 @@
+# How-To: Develop and F´ Library
+
+This guide will walk you through the structure and best practices in developing an F´ library.  F´ libraries are used to share modules (components, ports, topologies, etc.), toolchains, and platforms between F´ projects.  These libraries help with code reuse between projects by enabling direct sharing of F´ software.
+
+*Contents:*
+1. [F´ Library Structure](#flibrary-structure)
+2. [Required: `library.cmake`](#required-librarycmake)
+3. [Optional: Toolchain Folder and Toolchain Files](#optional-toolchain-folder-and-toolchain-files)
+4. [Optional: Platform Folder and Platform Files](#optional-platform-folder-and-platform-files)
+5. [Optional: F´ Module Directories](#optional-fmodule-directories)
+
+## F´ Library Structure
+
+In this section, you will learn about the expected structure of an F´ library. An F´ library must have a `library.cmake` and may have any of the following items:
+
+1. `cmake/toolchain` Folder and Toolchain Files
+2. `cmake/platform` Folder and Platform Files
+3. Module Directories and Modules (Components, Ports, Topologies, etc.)
```

Comment:
Can an example of Ports be included as well? A link can be made to [this file](https://github.com/nasa/fprime/blob/devel/Drv/TcpClient/TcpClient.fpp) as an example of how ports can be segmented across interfaces.

Reply:
I linked to our tutorials which cover how to develop items.